docs/primitives/AbstractStateTransition.md

```diff
@@ -91,7 +91,7 @@
 |--------------------------|------------------------|--------------------| -------------------------------- |
 | **options**              | Object                 | no                 |                                  |
 | **options.skipSignature**| Boolean[=false]        | no                 |                                  |
-| **options.encodedBuffer**| Boolean[=false]        | no                 |                                  |
+| **options.keepIdentifiers**| Boolean[=false]        | no                 |                                  |
```

Comment:
Maybe something like `keepIdentifiersAsObjects` or `convertIdentifiersToBuffers` would be a little bit more descriptive? For a person who isn't very familiar with the terminology, it may sound like literally keep them or remove them completely if you passed `false`

Reply:
Agree!

Reply:
But `Buffer` also `Object` :(